cfdm/mixin/propertiesdatabounds.py

```diff
@@ -1364,6 +1364,100 @@ def insert_dimension(self, position, inplace=False):
 
         return c
 
+    def nc_clear_hdf5_chunksizes(self, bounds=True, interior_ring=True):
+        """Clear the HDF5 chunking strategy for the data.
+
+        .. versionadded:: (cfdm) NEXTVERSION
+
+        .. seealso:: `nc_hdf5_chunksizes`, `nc_set_hdf5_chunksizes`,
+                     `{{package}}.read`, `{{package}}.write`
+
+        :Parameters:
+
+            bounds: `bool`, optional
+                If True, the default, then clear the HDF5 chunking
+                strategy from any bounds. If False then leave the
+                bounds chunking strategy unchanged.
+
+            interior_ring: `bool`
+                If True, the default, then clear the HDF5 chunking
+                strategy from a geometry interior ring variable. If
+                False then leave the geometry interior ring variable
+                chunking straegty unchanged.
+
+        :Returns:
+
+            `None` or `str` or `int` or `tuple` of `int`
+                The chunking strategy prior to being cleared, as would
+                be returned by `nc_hdf5_chunksizes`.
+
+        """
+        super().nc_clear_hdf5_chunksizes()
+
+        # Clear the bounds HDF5 chunks
+        if bounds:
+            bounds = self.get_bounds(None)
+            if bounds is not None:
+                bounds.nc_clear_hdf5_chunksizes()
+
+        # Clear the  interior_ring HDF5 chunks
+        if interior_ring:
+            interior_ring = self.get_interior_ring(None)
+            if interior_ring is not None:
+                interior_ring.nc_clear_hdf5_chunksizes()
+
+    def nc_set_hdf5_chunksizes(
+        self, chunksizes, bounds=True, interior_ring=True
+    ):
+        """Set the HDF5 chunking strategy.
+
+        .. versionadded:: (cfdm) NEXTVERSION
+
+        .. seealso:: `nc_hdf5_chunksizes`, `nc_clear_hdf5_chunksizes`,
+                     `{{package}}.read`, `{{package}}.write`
+
+        :Parameters:
+
+            {{hdf5 chunksizes}}
+                  Each dictionary key is an integer that specifies an
+                  axis by its position in the data array.
+
+            bounds: `bool`, optional
+                If True, the default, then apply the HDF5 chunking
+                strategy to the corresponding axes of any bounds. If
```

Comment:
Possibly not so - the parent data could be multidimensional. e.g. parent shape (73, 96) and  bounds shape (73, 96, 4).

(The interior ring variable will have on ly corresponding axis, though, as it can only exist on a 1-d coordinate construct.)

Reply:
Ah, that makes sense. Thanks for clarifying. Marking this one as resolved then, since no action is required.